### ISSUE TITLE
AVT 3 - Search and Form Component: VO does not announce the purpose of the button

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#2492]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed Description
With VoiceOver enabled, 
Tab to the Search field and enter some text
Tab to the X button, VO does not announce the purpose of the button.
Expected something like, "Clear Search"

**Comment by snidersd** [comment]
I just retested and confirmed this is no longer an issue since the X (close button) was removed from the component.
